### PR TITLE
Create pop windows

### PR DESCRIPTION
**Pull request by peteryeungtt**
In this project we needed to implement a popup window that includes the project details in both desktop and mobile in the project section.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -68,3 +68,212 @@ const storageObject = JSON.parse(localStorage.getItem('storageString'));
...
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+`;
+  const container = document.createElement('div');
+  container.innerHTML = htmlCode;
+  document.body.appendChild(container);
+}
```
Comment (ridwanediallo) [inline index.js]: - [ ] Hello there, @peteryeungtt. Congratulations on submitting your project for review. However, for some reason, the entire work section is not displaying, making it impossible to determine whether the popup modal is functioning properly. Please fix that problem and ensure that all of the project's requirements are met. :ok_hand: 😊

### Work section not displaying :point_down: 

Screenshot from 2023-05-18 18-48-48

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @peteryeungtt , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your PR has a good title. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (peteryeungtt) [reply index.js]: @ridwanediallo That is weird it is working on my end

Comment (peteryeungtt) [reply index.js]: @ridwanediallo 
image

Comment (peteryeungtt) [reply index.js]: image

Comment (peteryeungtt) [reply index.js]: @ridwanediallo I just clone my repo n checkout to branch "created_popups"
It is working fine on my end, I do not understand

Comment (peteryeungtt) [reply index.js]: @ridwanediallo I dont know what to do or troubleshoot because it is working on my end, I even tried to clone it 3 times

Comment (peteryeungtt) [reply index.js]: @ridwanediallo please check my live link:  https://peteryeungtt.github.io/My-Portfolio/
It is working properly.

File: styles.css [hunk 2]
```diff
@@ -883,3 +895,235 @@ input:focus {
...
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  backdrop-filter: blur(0.1rem);
+  transition: all 0.5s ease;
+  top: 0;
+}
+
```
Comment (Qoosim) [inline styles.css:927]: - [ ] It is required to produce a close replica of the figma design. Kindly check how your design displays on the mobile with how it should be on the figma. Some content of your modal on mobile are not properly visible. Kindly adjust your styling to make them visible as we have them on the Figma design.

### Your Design

your-mobile-popup

### Figma Design
temp-mobilepopup

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — APPROVED** [review]
Hi @peteryeungtt  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Meaningful commit messages💯
-  Project works as expected👍
-  Design looks like the Fgima design 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (ridwanediallo) [reply index.js]: - [ ] Hello there, @peteryeungtt. I apologize for all of the comments; I was away. I'm not sure what the problem was. When I run the project exactly as I received it yesterday, I get the same result. :point_down: 
Screenshot from 2023-05-19 12-27-43

What browser do you use?

**Pull request merged**